README.md

```diff
@@ -36,70 +36,90 @@ The Open Methane prior can be installed from source into a virtual environment w
 make virtual-environment
 ```
 
+The `Makefile` contains the set of commands to create the virtual environment.
+You can read the instructions out and run the commands by hand if you wish.
+
+### Input Data
+
+To download all the required input files, run:
+
+```console
+make download
```

Comment:
I replaced the typed commands with the make commands and added a note that the full commands can be found in Makefile. We can also do it the other way round